packages/mobile/src/store/store.ts

```diff
@@ -81,7 +81,12 @@ const createRootReducer = () =>
     shareToStoryProgress
   })
 
-const sagaMiddleware = createSagaMiddleware({ context: storeContext })
+const sagaMiddleware = createSagaMiddleware({
+  context: storeContext,
+  onError: (e) => {
+    console.error('Caught Saga Error', e, e.stack)
```

Comment:
we will get this in sentry now too. this is great!

Reply:
Sweet!! Is this a reasonable way to log it for readability in sentry?
I just wrote this for local debugging, but figured it's good to include

Reply:
this should be enough. they'll truncate our stack trace, but we'll learn quickly